tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-rc.yaml

```diff
@@ -27,6 +27,12 @@ spec:
         volumeMounts:
           - name: host-empty-vol
             mountPath: "/host/cache"
+          - mountPath: /tmp/results
+            name: hostpath-vol
+            readOnly: true
```

Comment:
Should we also add a read-write host mount to ensure that path is tested as well?

Reply:
Good call. I'll add readOnly and non readOnly for both.

Reply:
Updated to test both paths for emptyDir and hostPath